mp/src/game/client/clientmode_shared.cpp

```diff
@@ -823,13 +823,12 @@ int ClientModeShared::HandleSpectatorKeyInput( int down, ButtonCode_t keynum, co
 		else
 			engine->ExecuteClientCmd("glow_outline_effect_enable 1");
```

Comment:
You could avoid the redundant hard coding of the cvar & the engine dll indirection with:
```diff
-               if (glow_outline_effect_enable->GetBool())
-                       engine->ExecuteClientCmd("glow_outline_effect_enable 0");
-               else
-                       engine->ExecuteClientCmd("glow_outline_effect_enable 1");
                bool enabled = glow_outline_effect_enable->GetBool();
+               enabled = !enabled;
+               glow_outline_effect_enable->SetValue(enabled);
```